miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/api/ManifestApi.kt

```diff
@@ -17,5 +17,10 @@ internal interface ManifestApi {
 }
 
 internal data class ManifestEntity(
-    @SerializedName("manifest") val files: List<String>
+    @SerializedName("manifest") val files: List<String>,
+    @SerializedName("publicKeyId") val publicKeyId: String
```

Comment:
as `publicKeyId` is the same as the variable name you may not need the annotation there